frontend/public/components/monitoring/alerting.tsx

```diff
@@ -2379,40 +2407,29 @@ const PollerPages = () => {
   }, [alertsSource, dispatch]);
 
   return (
-    <Switch>
-      <Route path="/monitoring/(alerts|alertrules|silences)" exact component={AlertingPage} />
-      <Route path="/monitoring/alertrules/:id" exact component={AlertRulesDetailsPage} />
-      <Route path="/monitoring/alerts/:ruleID" exact component={AlertsDetailsPage} />
-      <Route path="/monitoring/silences/:id" exact component={SilencesDetailsPage} />
-      <Route path="/monitoring/silences/:id/edit" exact component={EditSilence} />
-    </Switch>
+    <Routes>
```

Comment:
Putting all of the routes in here will mean that the API pollers are now run for all of these pages. But the data is only used by some of the pages (those under the `PollerPages` component). Can we leave them separate as before?

Reply:
@kyoto could you specify why are those ? I can find `PollerPages` component in the console repo.

Reply:
The existing `PollerPages` component before this change (in frontend/public/components/monitoring/alerting.tsx) starts a poller to fetch the alerting rules from the Prometheus API and then renders the following routes:
```
    <Switch>
      <Route path="/monitoring/(alerts|alertrules|silences)" exact component={AlertingPage} />
      <Route path="/monitoring/alertrules/:id" exact component={AlertRulesDetailsPage} />
      <Route path="/monitoring/alerts/:ruleID" exact component={AlertsDetailsPage} />
      <Route path="/monitoring/silences/:id" exact component={SilencesDetailsPage} />
      <Route path="/monitoring/silences/:id/edit" exact component={EditSilence} />
    </Switch>
```
(These are pages that use the Prometheus alerting rules data.)

The problem is that this change adds various other routes to those rendered by `PollerPages`. That will cause the API poller will be run for those pages even though the data is not needed.